source/istio_virtualservice.go

```diff
@@ -97,6 +99,14 @@ func NewIstioVirtualServiceSource(
 		},
 	)
 
+	gatewayInformer.Informer().AddEventHandler(
+		cache.ResourceEventHandlerFuncs{
+			AddFunc: func(obj interface{}) {
+				log.Debug("gateway added")
+			},
+		},
+	)
```

Comment:
I suggest to either remove this log or provide a more detailed debug log.

Reply:
Sure, should I also change the other informers? I just replicated the other ones. I suggest removing all of them. Wdyt?

Reply:
@mloiseleur the eventhandler is needed because of the following behaviour on the go-client: https://github.com/kubernetes/client-go/issues/382, see also the comment above the first informer.

Reply:
ok, thanks.